ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-release-4.7.yaml

```diff
@@ -24,7 +24,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.14
```

Comment:
any reason not to go to 1.15?

though i don't think this really has any bearing on anything either, in terms of how the code is actually built.

Reply:
updated in current push